src/Neovim.jl

```diff
@@ -9,29 +9,32 @@ export Buffer, Tabpage, Window
 export reply_result, reply_error
 
 # types that have api methods (nvim itself + api defined types)
-abstract NvimObject
+abstract type NvimObject end
 
 include("client.jl")
 include("api_gen.jl")
 include("interface.jl")
 include("plugin_host.jl")
 
 # too inconvenient api to supply handler here?
-function nvim_connect(path::ByteString, args...)
+function nvim_connect(path::String, args...)
     s = connect(path)
     NvimClient(s, s, args...)
 end
 
 nvim_env(args...) = nvim_connect(ENV["NVIM_LISTEN_ADDRESS"], args...)
 
 function nvim_spawn(args...; cmd=`nvim --embed`)
-    output, input, proc = readandwrite(cmd)
-    (NvimClient(input, output, args...), proc)
+    # TODO(smolck): Just use the Process handle instead of p.in and p.out
+    #output, input, proc = readandwrite(cmd)
+    # (NvimClient(p, args...), proc)
```

Comment:
Can remove comments

Reply:
Fixed in https://github.com/bfredl/Neovim.jl/commit/97ae01b61b6696dcbf3ed8ccfd285658311cb256